docs/about.rst

```diff
@@ -0,0 +1,31 @@
+About
+=====
+
+Disclaimer
+----------
+
+Naggum don't aim to be yet another Common Lisp or Scheme or whatever
+implementation. Instead, we are trying to deliver a modern Lisp dialect that
+makes use of most of CLI benefits.
+
+Features
+--------
+
+- low level and high level features coupled together
```

Comment:
Unfortunately, all these functions and namespaces are already "low-level features of CLI". But I'll try to rephrase that.